webcompat/templates/shared/footer.html

```diff
@@ -16,7 +16,8 @@
             About
           </a>
         </li>
-        <li class="footer-item{% if request.url_rule.endpoint == 'contributors' %} active{% endif %}">
+        <li class="footer-item{% if request.url_rule.endpoint in 
+        ['contributors_build_tools', 'contributors_web_platform_research', 'contributors_organize_webcompat_events', 'contributors', 'contributors_report_bug', 'contributors_reproduce_bug', 'contributors_diagnose_bug', 'contributors_site_outreach'] %} active{% endif %}">
```

Comment:
Let's file a follow up bug to think of a way to do this a bit more simply, probably putting the list in python and passing into the template or something similar.